### PR TITLE
Document the download location of Linux, Windows and Mac OS X binaries

### PR DESCRIPTION
**Pull request by ankushagarwal**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Oh thanks! I actually like this! Sure, it doesn't fully address https://github.com/docker/docker/issues/7974, but I think this is a nice addition.

(Should we have a separate section for Windows? - Doesn't have to hold up this PR IMO)

Given that we currently don't have anything like this, and that it's useful; LGTM

/cc @moxiegirl 
/cc @ahmetalpbalkan for thoughts on the Windows section follow-up

**Comment by ankushagarwal** [comment]
I wanted to add Windows too, but I have no clue as to where those binaries are

**Comment by thaJeztah** [comment]
@ankushagarwal this comment mentions the URLs https://github.com/docker/docker/pull/11635#issuecomment-93823919

**Comment by ankushagarwal** [comment]
Okay. Maybe I (or someone else) can that in another PR as I have no context of Windows. Also I see that only 1.6.0 binaries are available for Windows and none prior to that.

**Comment by ahmetalpbalkan** [comment]
Replace Linux with Windows, add .exe to the end. :-)

**Comment by thaJeztah** [comment]
> Okay. Maybe I (or someone else) can that in another PR as I have no context of Windows. Also I see that only 1.6.0 binaries are available for Windows and none prior to that.

A note that Windows was added in 1.6 could do the trick there.

Also, I'm fine with this being added in a follow-up PR (don't have time myself soon, but perhaps someone else).

Thanks for doing this @ankushagarwal! LGTM, but the docs team will have the final say in this :)

**Comment by ankushagarwal** [comment]
@ahmetalpbalkan @thaJeztah : Added Windows locations as well, cause 
image

**Comment by ahmetalpbalkan** [comment]
Damn straight

**Comment by thaJeztah** [comment]
Haha! Thanks @ankushagarwal  can you please squash.... Never mind you just did. Thanks!

File: docs/sources/installation/binaries.md [hunk 1]
```diff
@@ -80,6 +80,8 @@ exhibit unexpected behaviour.
 
 ## Get the docker binary:
```
Comment (moxiegirl) [inline docs/sources/installation/binaries.md]: This heading needs a tweak if you don't mind when you are in there removing the colon.

File: docs/sources/installation/binaries.md [hunk 2]
```diff
@@ -80,6 +80,8 @@ exhibit unexpected behaviour.
...
 
+### Latest binary for Linux
```
Comment (moxiegirl) [inline docs/sources/installation/binaries.md]: Nice contribution!  Two heads in a row are called stacked heads --- meaning no intervening clarifying topic sentence.  I think all the downloads require setting the bit to execute; wasn't able to test Windows I don't have my Windows machine here.

No need to have a notes  in such short sections. I was able to bump up all the head levels making them more visible by putting the one paragraph up and breaking the stacked head mentioned above... other minor tweaks:

---> Markup here: https://gist.github.com/moxiegirl/b08dd0195f24920ff243

## Get a docker binary

You can download the latest release binary or a specific version. After downloading a binary file, you must set the file's execute bit to run it.

The latest binary is available only for Linux. You can download specific binary
versions for Linux, Mac OS X, and Windows. To get the list of stable release
version numbers from Github, view the `docker/docker` [releases
page](https://github.com/docker/docker/releases). 

### Get the latest binary

To download the latest binary for Linux:

    $ wget https://get.docker.com/builds/Linux/x86_64/docker-latest -O docker
    
To set the file's execute bit:

    $ chmod +x docker

If you have trouble downloading the binary, you can also get the smaller,
compressed release file:
[https://get.docker.com/builds/Linux/x86_64/docker-latest.tgz](https://get.
docker.com/builds/Linux/x86_64/docker-latest.tgz)

### Get a Linux binary

To download a specific version for Linux, use the
following URL syntax:

    https://get.docker.com/builds/Linux/i386/docker-<version>
    
    https://get.docker.com/builds/Linux/x86_64/docker-<version>

For example:

    https://get.docker.com/builds/Linux/i386/docker-1.6.0

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0.md5

    https://get.docker.com/builds/Linux/x86_64/docker-1.6.0.sha256


### Get an OS X binary

The Mac OS X binary is only a client. You cannot use it to run the `docker`
daemon. To download a specific Max OS X binary, use the following URL pattern:

    https://get.docker.com/builds/Darwin/i386/docker-<version>
    
    https://get.docker.com/builds/Darwin/x86_64/docker-<version>

For example:

    https://get.docker.com/builds/Darwin/i386/docker-1.6.0

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0.md5

    https://get.docker.com/builds/Darwin/x86_64/docker-1.6.0.sha256

###  Get a Windows binary
 
You can only download a Windows client binary for version `1.6.0` onwards.
Moreover, binary is only a client, you cannot use it to run the `docker` daemon.
To download a binary use the following pattern:

    https://get.docker.com/builds/Windows/i386/docker-<version>.exe
    
    https://get.docker.com/builds/Windows/x86_64/docker-<version>.exe

For example:

    https://get.docker.com/builds/Windows/i386/docker-1.6.0.exe

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe

You can also get the MD5 and SHA256 hashes:

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe.md5

    https://get.docker.com/builds/Windows/x86_64/docker-1.6.0.exe.sha256

**Comment by moxiegirl** [comment]
I didn't have time to test, I assume that Mac OS X and Windows get the daemon from `boot2docker` normally I'd mention that too.  If my guess without text, you an change the lead in para's for these sections as follows:

You can only download a Windows client binary for version `1.6.0` onwards.
Moreover, binary is only a client, you cannot use it to run the `docker` daemon. To run the daemon, use your Boot2Docker software. To download a binary use the following pattern:

File: docs/sources/installation/binaries.md (hunk 2 above)
Comment (ahmetalpbalkan) [inline docs/sources/installation/binaries.md]: @moxiegirl so what exactly have you changed? It's sort of difficult to take diff of 100 lines by eye.

**Comment by moxiegirl** [comment]
@ahmetalpbalkan I detailed the changes in my note. If you want a line by line diff. You can pull the fork and paste the gist appropriately.

**Comment by ahmetalpbalkan** [comment]
We can just have a note saying add .md5/.sha256 to URLs to get their hashes. I don't think there's a need to discuss the same thing over and over in 3 different sections.

> **Side note:** I always liked download page of golang https://golang.org/dl/ It's such a simple table (and given that ours are just binaries, it could look way simpler):
> image
> Just an idea, not really a comment to the PR.

**Comment by ankushagarwal** [comment]
@ahmetalpbalkan : Your suggestion is being discussed here : https://github.com/docker/docker/issues/7974

**Comment by ankushagarwal** [comment]
@moxiegirl  
> The latest binary is available only for Linux

Not exactly true. https://get.docker.com/builds/Darwin/x86_64/docker-latest and https://get.docker.com/builds/Windows/i386/docker-latest.exe also work

**Comment by ankushagarwal** [comment]
Addressed @moxiegirl's comments and made a few more edits. Here is the updated doc in gist : https://gist.github.com/ankushagarwal/b1bd7c973ddab19b18d2

**Comment by moxiegirl** [comment]
@ankushagarwal Thank you for the update! LGTM

**Pull request merged**